af_rest_appointment/models/af_appointment.py

```diff
@@ -62,39 +64,64 @@ def _generate_headers(self, af_environment, af_system_id, af_tracking_id):
         return get_headers
 
     # /bookable-occasions
-    def get_occasions(self, from_date, to_date, appointment_channel, appointment_type, max_depth = False, appointment_length = False, location_code = False, profession_id = False, employee_user_id = False):
-        client_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_id')
-        client_secret = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_secret')
-        af_environment = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_environment')
-        af_port = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_port')
-        af_url = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_url')
-        af_system_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_system_id')
-
-        if not (af_url or af_port or client_id or client_secret or af_environment or af_system_id):
+    def get_occasions(
+            self,
+            from_date,
+            to_date,
+            appointment_channel,
+            appointment_type,
+            max_depth=False,
+            appointment_length=False,
+            location_code=False,
+            profession_id=False,
+            employee_user_id=False):
+        client_id = self.env['ir.config_parameter'].sudo(
```

Comment:
This is almost unreadable. could you loook into any other way of getting these lines to go through Travis?